version.tf

```diff
@@ -3,13 +3,13 @@
 #####################################################
 
 terraform {
-  required_version = ">= 1.1.0"
+  required_version = ">= 1.3.0"
   required_providers {
     # Use "greater than or equal to" range in modules
     # tflint-ignore: terraform_unused_required_providers
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.49.0"
+      version = "=1.50.0"
```

Comment:
Just to confirm, you defiantly want to lock into exact version here? Previously `>=` was used to give flexability, but maybe it doesn't matter since the solutions (examples) are all locked into exact version?

Reply:
Yes, because this solution has been tested with a particular version. Keeping it uniform across all submodules and the root will upgrade when making new releases. Dont want something else to crash. :)